dashboard/config/scripts/levels/allthethings design mode elements.level

```diff
@@ -167,4 +167,4 @@
   }
 }]]></config>
   <blocks/>
-</Applab>
\ No newline at end of file
+</Applab>
```

Comment:
do we have to worry about running into problems modifying .level files from outside LB?